railties/lib/rails/test_unit/minitest_plugin.rb

```diff
@@ -0,0 +1,14 @@
+require "minitest"
+require "rails/test_unit/reporter"
+
+def Minitest.plugin_rails_init(options)
+  self.reporter << Rails::TestUnitReporter.new(options[:io], options)
+  if $rails_test_runner && (method = $rails_test_runner.find_method)
```

Comment:
This plugin is here for two things:
- Add the `TestUnitReporter` , which outputs: 
```
Failed tests:

bin/rails test /Users/arthurnn/dev/railstest/myapp/test/models/comment_test.rb:8
```
- Add a filter option to minitest. So we can find the `method` that we want to run if the runner had a line argument specified, and only run that individual test.